template.tpl

```diff
@@ -744,6 +854,40 @@ if (data.feature === 'conversionTracker') {
   }
 
   return conf;
+} else if (data.feature === 'vod') {
+  
+  const conf = {
+    orgId: data.bambuserOrgId,
+    containers: data.vodContainers
+  };
+  const launch = function () {
+    if (queryPermission('access_globals', 'execute', 'injectBambuserPlayer')) {
+      callInWindow('injectBambuserPlayer', conf);
+      data.gtmOnSuccess();
+    }
```

Comment:
Don't know if this permission can ever be false? But if so, maybe we should call `data.gtmOnFailure()`? 🤔